sigle/src/modules/editor/TipTapEditor.tsx

```diff
@@ -94,6 +96,7 @@ interface TipTapEditorProps {
   editable?: boolean;
 }
 
+//eslint-disable-next-line
```

Comment:
I feel like this one shouldn't be committed, from my understanding it does not break the CI nor on my VScode so my guess is that your vscode config is somehow creating some issues there. Could you please remove it ?